src/v1/subscriber_client_config.json

```diff
@@ -86,7 +86,7 @@
           "retry_params_name": "messaging"
         },
         "StreamingPull": {
-          "timeout_millis": 60000,
```

Comment:
@alexander-fenster I think this value is generated, so anytime we update gapics it gets reverted back to 1 minute. I know it seems like a reasonable amount, by when applying grpc deadlines to streams, it actually kills the stream after the designated time regardless of whether or not a connection was established.

That being said, I bumped it to 15 minutes (I think that's where it was before. Is there something on the gapic side of things that could be changed so we don't have to update this value?

Reply:
Yes, the generator takes it from here https://github.com/googleapis/googleapis/blob/b92c84b696a22dbecbcf53c760bc9195af4406b4/google/pubsub/v1/pubsub_gapic.yaml#L259 and this file is replicated from internal source control. I will make a request to set it to 900000 - is that what you need?

Reply:
It's now updated to 900000. https://github.com/googleapis/googleapis/blob/d90956b8adf6c820969457d848db833328076879/google/pubsub/v1/pubsub_gapic.yaml#L259

Reply:
@callmehiphop is there there a way to pass this in options instead of getting it from the internal config only.